docs/en/wallet-attestation.rst

```diff
@@ -559,7 +559,20 @@ The details of the revocation mechanism used by the Wallet Provider as well as t
 During the *Wallet Instance initialization and registration* phase the Wallet Provider MAY associate the Wallet Instance with a specific User, subject to obtaining the User's consent. The Wallet Provider MUST evaluate the operating system and general technical capabilities of the device to check compliance with the technical and security requirements and to produce the Wallet Instance metadata.
 When the User consents to being linked with the Wallet Instance, they gain the ability to directly request Wallet revocation from the Wallet Provider, and it also allows the Wallet Provider to revoke the Wallet Instance associated with that User.
 
+The following situations could occur:
 
+- The smartphone is lost;
+- The smartphone has been compromised (e.g., a malicious actor gains control of the smartphone);
+- The smartphone has been reset to factory settings;
+- Other scenarios where the owner can no longer access their Wallet Instance.
+
+If any of these situations occur, the owner **MUST** revoke the Wallet Instance.
```

Comment:
```suggestion
If any of these situations occur, the owner **MUST** revoke the Wallet Instance.
```

please define the "owner"

Reply:
I would suggest to use 'holder' in every occurrence of 'owner' .
Holder would be defined as: "User that has activated and has full control of an instance of the EUDI wallet and the related private keys".
Some reference of this can be seen in art. 6a, point 7 of the amending regulation